src/histolab/slide.py

```diff
@@ -26,14 +26,30 @@
 import PIL
 from skimage.measure import find_contours
 
-from .exceptions import LevelError
+from .exceptions import HistolabException, LevelError
 from .filters.compositions import FiltersComposition
 from .masks import BinaryMask
 from .tile import Tile
 from .types import CoordinatePair
 from .util import lazyproperty
 
+try:
+    import large_image
+    from io import BytesIO
+
+    LARGEIMAGE_IS_INSTALLED = True
+    LARGEIMAGE_INSTALL_PROMPT = ""
```

Comment:
Sure thing. These two constants have values that change depending on whether or not ``large_image`` is installed. This is necessary because the try-except import block is at the top of the file (as it should) and depending on whether or not ``large_image`` is available the error message should be different (see my response to your other comment below).

Reply:
OK I removed it and used more specific error messages instead.